focus-areas/ecosystem/social-currency-metric-system.md

```diff
@@ -118,7 +118,7 @@ Community member comments are available from trace data. The SCMS ideally import
 _Points of destruction_: Detailed data is destroyed after _xx_ months has passed. Quantitative calculations are kept for up to 250 weeks per GDPR compliance. Data older than 250 weeks becomes archived data you cannot manipulate but can see. Users can negotiate the primary statistic.
 
 
-## References
+## Resources
```

Comment:
Good catch!

Thank you!

Reply:
I'm leaving this open, I realize I'm more than 15 days in by now and so I'm likely looking for cues (taking in details) so spotting (#not-a-douche) is just me distracting myself (and so thanks for being awesome).

Reply:
I checked our template and it is supposed to be "References"

https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md

Reply:
I think I spotted "Resources" in [wg-diversity-inclusion#243](https://github.com/chaoss/wg-diversity-inclusion/pull/243) and likely others that indicated "v2" templates — sorry I did not know/remember checking the template itself — so is it okay then to open a PR to changes "Resources" to "References" (that felt more appropriate in PR I mentioned BTW but wasn't sure it was time to debate stylistic matters there 👍).

Reply:
Yes, it's appropriate to open PRs to fix any templating mistakes we have in our metrics -- in fact, that would be a huge help.